platform-coop-toolkit/coop-library-framework
# Add archival link fields (resolves #6, resolves #54, resolves #72)

## Description

This PR adds the ability to include one or more links to archival copies of a resource on [Perma.cc](https://perma.cc) and the [Wayback Machine](https://web.archive.org). Field contents will be validated to ensure that they contain relevant URLs.

Additionally, `npm run localize` now ignores PHP files as they can be localized using Polylang (see #82).

## Steps to test

1. Add one or more Perma.cc and Wayback Machine URLs to a resource.
2. Add erroneous links to the Perma.cc and Wayback Machine fields.

**Expected behavior:** Step 1 should result in a successful save of the resource. Step 2 should flag the error for the user.

## Additional information

- https://wiki.fluidproject.org/display/fluid/Resource+preservation+research

## Related issues

- Resolves #6
- Resolves #54 
- Resolves #72
